jsb.js

```diff
@@ -379,118 +368,26 @@ jsb = {
          * We need to use concat, because otherwise the returned array would
          * change as soon as we remove the element's className.
          */
-        var raw_dom_elements = null;
-        if (dom_element.getElementsByClassName) {
-            raw_dom_elements = dom_element.getElementsByClassName(this.prefix);
```

Comment:
Please put both back in.

Am Sonntag, 10. April 2016 schrieb Lars Laade :

> In jsb.js <https://github.com/DracoBlue/jsb/pull/29#discussion_r59131834>:
>
> > @@ -379,118 +368,26 @@ jsb = {
> >           * We need to use concat, because otherwise the returned array would
> >           * change as soon as we remove the element's className.
> >           */
> > -        var raw_dom_elements = null;
> > -        if (dom_element.getElementsByClassName) {
> > -            raw_dom_elements = dom_element.getElementsByClassName(this.prefix);
>
> So you are ok with this change or should i put the fallback back in? What
> is with attachEvent (ie8 and lower would only need this), i removed
> <https://github.com/DracoBlue/jsb/pull/29/commits/23e96ec280e94ebd10a5485dd341e9063123298e>
> this, too?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/DracoBlue/jsb/pull/29/files/c28d72872894e571c627926e603396650502fce9#r59131834>
>


-- 

  http://dracoblue.net